source/_integrations/tuya.markdown

```diff
@@ -116,8 +116,8 @@ Click the created project to enter the `Project Overview` page and get the `Auth
 
 "1106: permission deny":
   description: >
-    - App account not linked with cloud project: On the [Tuya IoT Platform](https://iot.tuya.com/cloud/), you have linked devices by using Tuya Smart or Smart Life app in your cloud project. For more information, see [Link devices by app account](https://developer.tuya.com/en/docs/iot/Platform_Configuration_smarthome?id=Kamcgamwoevrx&_source=7a356dd493196a01bb9021b7680a2a45#title-3-Link%20devices%20by%20app%20account).
-
+    - App account not linked with cloud project: On the [Tuya IoT Platform](https://iot.tuya.com/cloud/), you have linked devices by using Tuya Smart or Smart Life app in your cloud project. For more information, see [Link devices by app account](https://developer.tuya.com/en/docs/iot/Platform_Configuration_smarthome?id=Kamcgamwoevrx&_source=7a356dd493196a01bb9021b7680a2a45#title-3-Link%20devices%20by%20app%20account). Error might occur when Tuya Smart or Smart Life account was created with Google, to solve this problem change your password in Tuya Smart or Smart Life app.
```

Comment:
The piece you are trying to add, is literally listed just one point below, where it states that social logins may not work.